install-kubeadm.sh

```diff
@@ -2,6 +2,18 @@ apt-get update
 apt-get install -y docker.io
 sudo usermod -a -G docker vagrant
 
+sudo sed -i "/$(hostname)/d" /etc/hosts
+sudo echo "
+192.168.50.2 k8s-master
+192.168.50.3 k8s-worker-0
+192.168.50.4 k8s-worker-1
+192.168.50.1 k8s-lb
+" >> /etc/hosts
+
+sysctl net.bridge.bridge-nf-call-iptables=1
+swapoff -a
+sudo sed -i.bak '/ swap / s/^\(.*\)$/#\1/g' /etc/fstab
```

Comment:
This is just too clever for me to understand. Of course, I could try the command myself, but I would rather have a small comment explaining it.

Reply:
First line is required by weavenet as documented here : 
https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network (click weave net)

Second line is your own, turning off swap (only until reboot)
Last list inserts a "#" in /etc/fstab where the swap partition is, turning off swap permantly, also after a reboot. swapoff -a does not.

Reply:
Thanks, that is _exactly_ what I need as comments in the file :)